### PR TITLE
Fix boost installation 

### PR DESCRIPTION
**Pull request by meiravgri**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -237,6 +237,7 @@ CMAKE_FILES+= \
 endif
 
 #----------------------------------------------------------------------------------------------
+BOOST_DIR ?= $(ROOT)/.install/boost
```
Comment (GuyAv46) [inline Makefile:240]: I think this is ideal for CI but not for local development. IMO we should keep the default empty value, especially as this is a non-existing directory and is not ignored by `.gitignore`

Comment (meiravgri) [reply Makefile:240]: Then passing BOOST_DIR is mandatory...
I think it's a good practice to have a default value, allowing you to just run the script and go.
And we can add .install/boost_* to .gitignore :)

**Review by oshadmi — DISMISSED** [review]

File: Makefile [hunk 2]
```diff
@@ -237,7 +238,12 @@ CMAKE_FILES+= \
 endif
 
 #----------------------------------------------------------------------------------------------
-BOOST_DIR ?= $(ROOT)/.install/boost
+ifeq ($(wildcard $(ROOT)/.install/boost),)
```
Comment (GuyAv46) [inline Makefile:241]: we still want to allow passing an explicit `BOOST_DIR` value no? Should we add the condition that it's not defined?

Comment (meiravgri) [reply Makefile:241]: oops

**Review by GuyAv46 — APPROVED** [review]

**Pull request merged**